holmes/plugins/toolsets/grafana/grafana_api.py

```
@@ -11,32 +11,48 @@
+        health_urls.append(
+            f"{config.url}/api/datasources/uid/{config.grafana_datasource_uid}/health"
+        )
+
+    health_urls.append(f"{get_base_url(config)}/{config.healthcheck}")
+    g_headers = build_headers(api_key=config.api_key, additional_headers=None)
+
+    error_msg = ""
+    for url in health_urls:
+        try:
+            _try_health_url(url, g_headers)
+            return True, ""
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)